stdlib/option.mc

```diff
@@ -1,4 +1,74 @@
 type Option
 
 con Some : Dyn -> Option
-con None : Option
\ No newline at end of file
+con None : Option
+
+-- Returns `true` if the option is a `Some` value.
+let isNone: Option -> Bool = lam o.
+  match o with None then
+    true
+  else
+    false
+
+-- Returns `true` if the option is a `None` value.
+let isSome: Option -> Bool = lam o.
+  match o with Some t then
+    true
+  else
+    false
+
+-- Returns `true` if the option contains the specified value.
+let contains: Option -> Dyn -> Bool = lam o. lam v.
+  match o with Some t then
+    if eqi t v then
```

Comment:
Woops, the equality check here should not work.

Reply:
Maybe provide a predicate instead? (cf. `find` in `seq.mc`)